test/modules/billing/services/supplementary-billing-service/data-service.test.js

```diff
@@ -26,6 +25,14 @@ const batchId = uuid()
 
 const { createTransaction } = require('./test-helpers')
 
+const pick = (obj, props) => {
```

Comment:
I chose to add a function here for pick otherwise I would have had to duplicate the keys which would have been an additional 50 lines of code (since pick is used twice in this file)

Reply:
In the test where `pick()` is used, if we refactor how it is doing the assertion we can remove the need to have a `pick()` function.

```javascript
// Do this
for (const key of matchingKeys) {
  expect(transactions[0][key]).to.equal(createdTransaction[key])
}

// instead of this
expect(pick(transactions[0], matchingKeys)).to.equal(pick(createdTransaction, matchingKeys))
```